src/main/java/pl/edu/mimuw/BankAction.java

```diff
@@ -3,17 +3,23 @@
 import java.sql.Timestamp;
 
 public abstract class BankAction {
+  protected double totalAmount;
+  protected Timestamp startTimestamp;
+  protected Timestamp endTimestamp;
 
```

Comment:
raczej należałoby dodać tutaj również konstruktor inicjalizujący te pola, tak by zapewnić, że każda podklasa je zianicjalizuje
 robimy to przez 
```
protected BankAction(pola...) {
 ...inicalizacja
}
```
a w podklasach wtedy trzeba zawolac 
```
super(pola...)
```
w każdym konstruktorze